README.md

```diff
@@ -192,6 +192,32 @@ Installing from GitHub is considered developer mode and it's documented in
   0 */2 * * * plextraktsync
   ```
 
+* Instead of cron, a docker scheduler like [Ofelia](https://github.com/mcuadros/ofelia/) can also be used to run the script at set intervals.
+
+  A docker-compose example with a 6h interval:
+  
+  ```
+  services:
+    scheduler:
+      image: mcuadros/ofelia:latest
+      container_name: scheduler
+      depends_on:
+        - ptsync
+      command: daemon --docker
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock:ro
+      labels:
+        ofelia.job-run.my_sync.schedule="@every 6h"
+        ofelia.job-run.my_sync.container="ptsync"
+    ptsync:
```

Comment:
the main example uses "plextraktsync", use the same here for consistency.

Reply:
agreed